qa/workunits/rados/test_envlibrados_for_rocksdb.sh

```diff
@@ -68,7 +68,11 @@ echo "Compile rocksdb"
 if [ -e rocksdb ]; then
 	rm -fr rocksdb
 fi
-git clone https://github.com/facebook/rocksdb.git --depth 1
+
+pushd $(dirname /home/ubuntu/cephtest/clone.client.0/qa/workunits/rados/bash.sh)/../../../
+git submodule update --init src/rocksdb
```

Comment:
why not just
```
cd src/rocksdb
```

and start compiling and testing it?

Reply:
Didn't want to leave any artefacts for any test that may run after this one.